templates/plans/index.hamlet

```diff
@@ -0,0 +1,13 @@
+$forall plan <- plans
+  <div .plan .col-md-4>
+    <div .panel.panel-default>
+      <div .panel-heading>
+        <h3 .panel-title>#{show plan}
+      <div .panel-body>
+        ^{planFeatures plan}
+
+<h2>Why Pay?
+
+<p>Carnival will never contain ads
+<p>Carnival will never track your behavior
```

Comment:
Maybe instead of a bulleted list, we should have a paragraph explaining the rationale of having a paid subscription to directly support the service rather than resorting to ads and shifty behavior to monetize after the fact?